sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_models.hpp

```diff
@@ -970,4 +970,86 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
     IssuerProperties Properties;
   };
 
+  /**
+   * @brief A certificate operation is returned in case of asynchronous requests.
+   *
+   */
+  struct CertificateOperationProperties
```

Comment:
Make all of this fields be public members of the `CreateCertificateOperation`.

Then, a customer can see all this info from the Operation after each poll() call